backend/database/models/organization.py

```diff
@@ -0,0 +1,14 @@
+from ..core import db, CrudMixin
+from backend.database.models._assoc_tables import organization_user
+
+
+class Organization(db.Model, CrudMixin):
+    id = db.Column(db.String, primary_key=True)
+    name = db.Column(db.Text)
+    url = db.Column(db.Text)
+    contact_email = db.Column(db.Text)
+    reported_incidents = db.relationship(
+        'Incident', backref='organization', lazy="select")
```

Comment:
The backref here should still be source.

Reply:
If we're renaming `Source` to `Organization` on the back end for consistency with the front end, the database should also reflect that for consistency.